tests/cases/fourslash/completionsGeneratorFunctions.ts

```diff
@@ -0,0 +1,22 @@
+/// <reference path="fourslash.ts" />
+
+////function /*a*/ ;
+////function* /*b*/ ;
+////interface I {
+////    abstract baseMethod(): Iterable<number>;
+////}
+////class C implements I {
+////    */*c*/ ;
+////    public */*d*/
+////}
+////const o: I = {
+////    */*e*/
+////};
+////1 * /*f*/
+
+verify.completions(
+    { marker: ["a", "b"], exact: undefined, isNewIdentifierLocation: true },
+    { marker: ["c", "d"], exact: ["baseMethod"], isNewIdentifierLocation: true },
```

Comment:
Need to check keyword completions as well?

Reply:
`exact` ensures that there are no other completions we forgot to list.